sdk/servicebus/Microsoft.Azure.ServiceBus/src/ServiceBusConnectionStringBuilder.cs

```diff
@@ -221,6 +222,11 @@ public string SasToken
         /// <remarks>Defaults to 1 minute.</remarks>
         public TimeSpan OperationTimeout { get; set; } = Constants.DefaultOperationTimeout;
 
+        /// <summary>
+        /// Enables Azure Active Directory Managed Identity authentication when set to 'Managed Identity'
+        /// </summary>
+        public string Authentication { get; set; }
```

Comment:
>Authentication [](start = 22, length = 14)

Curious as to why this is a string? As compared to other options of - Enum or bool

Reply:
I'd suggest the enum approach, personally.  Asking consumers to pass a magic string seems like it would offer a challenge to discovering usable values via intellisense.

Reply:
In the case that you're set on a string, maybe consider adding a pseudo-enumeration to allow discovery and help prevent typos, then?  Something like:

```c#
public static class AuthenticationType
{
    public static const ManagedIdentity = "Managed Identity";
}
```

That also gives you the advantage of being able to throw doc comments on the type and member to guide the user experience.

Reply:
Is it expected to extend methods of authentication in the near future?

Reply:
This is another area that I've not seen a response and have concerns about user experience.  I don't want to block on it, as it is consistent with the direction that the Event Hubs client went, but I would love to see some thoughts after usability testing.